index.js

```diff
@@ -58,13 +60,34 @@ function handleRTCMessage(client, payload)
   }
 }
 
+getDisabledDefault = function(field)
+{
+  // TODO - have a settings.json.template type thing
```

Comment:
@JohnMcLear You mentioned this offhand. Is this for the future when we move to plugin-specific settings files, or is this something I should worry about here?